README.md

```diff
@@ -1,32 +1,30 @@
 # Hyperon DAS
 
-Hi! This package is a query engine API for Distributed AtomSpace (DAS). When is possible execute queries using **[Pattern Matcher](https://wiki.opencog.org/w/The_Pattern_Matcher)**
+A data manipulation API for Distributed Atomspace (DAS). It allows queries with pattern matching capabilities and traversal of the Atomspace hypergraph.
 
 ## Table of Contents
```

Comment:
After fixing all the comments in this review, the table of contents will be very short so this section can be removed.